### PR TITLE
add refernece to fastlane support

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**JIRA**
https://issues.jboss.org/browse/AGDIGGER-213

**Changes**
Adding reference to the fastlane support provided by aerogear digger. The user needs to include the Jenkinsfile and folders in the github folder so pointing the there rather than duplicating docs.

@pmacko1 @finp

### CODE REVIEW

File: docs/digger/developer/jenkinsfile-fastlane.asciidoc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+== Fastlane Support
+
```
Comment (finp) [inline docs/digger/developer/jenkinsfile-fastlane.asciidoc]: @laurafitzgerald  might be an idea to link to https://fastlane.tools/ ?

Comment (laurafitzgerald) [reply docs/digger/developer/jenkinsfile-fastlane.asciidoc]: @finp done :)

**Comment by pmacko1** [comment]
@laurafitzgerald could you rebase against Build-farm-2, so the changes are more visible, please?

**Comment by laurafitzgerald** [comment]
@pmacko1 done. I have rebased against master (as this is the destination of this pr) and the change is now more visible. However there are many extra commits here so will will open a new pr for this one.

**Pull request closed without merging**